Kitsune/ClubPenguin/Login.php

```diff
@@ -27,14 +27,15 @@ protected function handleLogin($socket) {
 		$password = Packet::$Data['body']['login']['pword'];
 		
 		if($penguin->database->usernameExists($username) === false) {
+			password_verify('KitsuneTimingAttack', '$2y$12$5hf7vOtBjsNJf6oZSsnyXOlcLTRblTwSg550SR0ohxVUkItmXhMM6'); //Mitigate timing attacks
```

Comment:
Ok, I see now why it's implemented here. What data does this addition prevent an attacker from gathering?